include/core/mortar_coupling_manager.h

```diff
@@ -303,10 +316,13 @@ class MortarManager
 
   /**
    * @brief Returns the normal vector for the quadrature points
+   * @param[in] rad Angular coordinate of cell center
+   * @param[out] result Normal vectors of the cell quadrature points
```

Comment:
```suggestion
   *
   * @return result Normal vectors of the cell quadrature points
```

param[out] is better suited when you pass a variable (say a reference) and the function modifies the content of that references. If you are creating something and returning something, it is better to use the @return flag here :)